src/routes/pageList.ts

```diff
@@ -1,6 +1,9 @@
 export async function getPages(
-	metaGlob: Array<string>
-): Promise<Array<{ [key: string]: string; filename: string; path: string; title?: string }>> {
```

Comment:
The `[key: string]: string` is still needed for building the hierarchy of recipes (a `children` key is injected later)